diff --git a/node_modules/@react-navigation/core/lib/typescript/src/types.d.ts b/node_modules/@react-navigation/core/lib/typescript/src/types.d.ts
index 01c3f8b..45751c9 100644
--- a/node_modules/@react-navigation/core/lib/typescript/src/types.d.ts
+++ b/node_modules/@react-navigation/core/lib/typescript/src/types.d.ts
@@ -63,6 +63,9 @@ export declare type EventMapCore<State extends NavigationState> = {
         };
         canPreventDefault: true;
     };
+    tabPress: {
+        data: undefined
+    };
 };
 export declare type EventArg<EventName extends string, CanPreventDefault extends boolean | undefined = false, Data = undefined> = {
     /**
