diff --git a/node_modules/lottie-react-native/lib/typescript/index.d.ts b/node_modules/lottie-react-native/lib/typescript/index.d.ts
index 111e0c1..096c400 100644
--- a/node_modules/lottie-react-native/lib/typescript/index.d.ts
+++ b/node_modules/lottie-react-native/lib/typescript/index.d.ts
@@ -1,3 +1,4 @@
 import { LottieView } from './LottieView';
+export * from './LottieView.types';
 export default LottieView;
 //# sourceMappingURL=index.d.ts.map
\ No newline at end of file
diff --git a/node_modules/lottie-react-native/src/index.tsx b/node_modules/lottie-react-native/src/index.tsx
index 15eb4e3..5f52403 100644
--- a/node_modules/lottie-react-native/src/index.tsx
+++ b/node_modules/lottie-react-native/src/index.tsx
@@ -1,2 +1,3 @@
 import { LottieView } from './LottieView';
+export * from './LottieView.types';
 export default LottieView;
